system/handlers/formbuilder/item-types/Date.cfc

```diff
@@ -13,13 +13,15 @@ component {
 
 			switch( args.relativeOperator ) {
 				case "lt":
-					args.maxDate = DateAdd( 'd', -1, Now() );
+					var diff = val(args.offset) ? val(-args.offset) : -1;
```

Comment:
I'm not sure of the default `-1` here (same for the "gt" offset). I would have thought:

```
if ( Val( args.offset ) ) {
    args.maxDate = DateAdd( 'd', Val( -args.offset ), Now() );
}
```
Can you check this?